### PR TITLE
V11: add test flakyness reporting

### PR DESCRIPTION
**Pull request by Zeegaan**
# Notes
- Bumped `playwright-testhelpers` package version.
- Added `CommitId` & `AgentOs` as variables on the build pipeline
- Added reporting to the before each loop of all tests.

### CODE REVIEW

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/recycleBin.spec.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ import {ContentBuilder, DocumentTypeBuilder} from "@umbraco/json-models-builders
 
 test.describe('Recycle bin', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/recycleBin.spec.ts:8]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

**Comment by andr317c** [comment]
Looks good!

But there is missing a semicolon after each report in the beforeEach

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Branding/logo.spec.ts [hunk 2]
```diff
@@ -3,10 +3,8 @@ import { expect } from "@playwright/test";
 
 test.describe('Umbraco Logo Information', () => {
 
-  test.beforeEach(async ({ page, umbracoApi }) => {
-    // TODO: REMOVE THIS WHEN SQLITE IS FIXED
-    // Wait so we don't bombard the API
-    await page.waitForTimeout(1000);
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Branding/logo.spec.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/content.spec.ts [hunk 3]
```diff
@@ -10,7 +10,8 @@ import {
 
 test.describe('Content tests', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/content.spec.ts:14]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/recycleBin.spec.ts (hunk 1 above)
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/recycleBin.spec.ts:8]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/routing.spec.ts [hunk 4]
```diff
@@ -11,7 +11,8 @@ test.describe('Routing', () => {
   const grandChildNodeName = "Grandchild";
   const rootDocTypeName = "Test document type";
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/routing.spec.ts:15]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/DataTypes/dataTypes.spec.ts [hunk 5]
```diff
@@ -8,7 +8,8 @@ import {
 
 test.describe('DataTypes', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/DataTypes/dataTypes.spec.ts:12]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/DataTypes/textBoxVariation.spec.ts [hunk 6]
```diff
@@ -4,7 +4,8 @@ import {ContentBuilder, DocumentTypeBuilder, DomainBuilder} from "@umbraco/json-
 
 test.describe('Vary by culture for TextBox', () => {
 
-    test.beforeEach(async ({page, umbracoApi, umbracoUi}) => {
+    test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+        await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/DataTypes/textBoxVariation.spec.ts:8]:
```suggestion
        await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/HelpPanel/systemInformation.spec.ts [hunk 7]
```diff
@@ -5,7 +5,8 @@ test.describe('System Information', () => {
   const enCulture = "en-US";
   const dkCulture = "da-DK";
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/HelpPanel/systemInformation.spec.ts:9]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Languages/languages.spec.ts [hunk 8]
```diff
@@ -3,7 +3,8 @@ import {expect} from "@playwright/test";
 
 test.describe('Languages', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Languages/languages.spec.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Members/memberGroups.spec.ts [hunk 9]
```diff
@@ -2,7 +2,8 @@ import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
 
 test.describe('Packages', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Members/memberGroups.spec.ts:6]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/ModelsBuilder/modelsbuilder.spec.ts [hunk 10]
```diff
@@ -6,7 +6,8 @@ import {
 
 test.describe('Modelsbuilder tests', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/ModelsBuilder/modelsbuilder.spec.ts:10]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Packages/packages.spec.ts [hunk 11]
```diff
@@ -7,7 +7,8 @@ test.describe('Packages', () => {
   const packageName = "TestPackage";
   const rootDocTypeName = "Test document type";
   const nodeName = "1) Home";
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Packages/packages.spec.ts:11]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/dataType.spec.ts [hunk 12]
```diff
@@ -3,8 +3,9 @@ import {expect} from "@playwright/test";
 import {LabelDataTypeBuilder} from "@umbraco/json-models-builders";
 
 test.describe('Data Types', () => {
-  
-  test.beforeEach(async ({page, umbracoApi}) => {
+
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/dataType.spec.ts:8]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/documentTypes.spec.ts [hunk 13]
```diff
@@ -3,7 +3,8 @@ import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
 import {DocumentTypeBuilder} from "@umbraco/json-models-builders";
 
 test.describe('Document types', () => {
-  test.beforeEach(async ({ page, umbracoApi }) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/documentTypes.spec.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/languages.spec.ts [hunk 14]
```diff
@@ -2,7 +2,8 @@ import {test, ApiHelpers, UiHelpers, ConstantHelper} from '@umbraco/playwright-t
 import {expect} from "@playwright/test";
 
 test.describe('Languages', () => {
-  test.beforeEach(async ({ page, umbracoApi }) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/languages.spec.ts:6]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/macro.spec.ts [hunk 15]
```diff
@@ -3,7 +3,8 @@ import {PartialViewMacroBuilder} from "@umbraco/json-models-builders";
 
 test.describe('Macros', () => {
 
-  test.beforeEach(async ({ page, umbracoApi }) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/macro.spec.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/mediaTypes.ts [hunk 16]
```diff
@@ -3,7 +3,8 @@ import {expect} from "@playwright/test";
 
 test.describe('Media types', () => {
 
-  test.beforeEach(async ({ page, umbracoApi }) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/mediaTypes.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/memberTypes.spec.ts [hunk 17]
```diff
@@ -1,7 +1,8 @@
 import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
 
 test.describe('Member Types', () => {
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/memberTypes.spec.ts:5]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/partialViewMacroFiles.spec.ts [hunk 18]
```diff
@@ -4,7 +4,8 @@ import {PartialViewMacroBuilder} from "@umbraco/json-models-builders";
 
 test.describe('Partial View Macro Files', () => {
 
-    test.beforeEach(async ({page, umbracoApi}) => {
+    test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+        await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/partialViewMacroFiles.spec.ts:8]:
```suggestion
        await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/partialViews.spec.ts [hunk 19]
```diff
@@ -4,7 +4,8 @@ import {PartialViewBuilder} from "@umbraco/json-models-builders";
 
 test.describe('Partial Views', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/partialViews.spec.ts:8]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/relationTypes.spec.ts [hunk 20]
```diff
@@ -3,7 +3,8 @@ import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
 
 test.describe('Relation Types', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/relationTypes.spec.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/scripts.spec.ts [hunk 21]
```diff
@@ -3,7 +3,8 @@ import {expect} from "@playwright/test";
 import {ScriptBuilder} from "@umbraco/json-models-builders";
 
 test.describe('Scripts', () => {
-  test.beforeEach(async ({ page, umbracoApi }) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/scripts.spec.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/stylesheets.spec.ts [hunk 22]
```diff
@@ -7,7 +7,8 @@ test.describe('Stylesheets', () => {
   const name = "TestStylesheet";
   const fileName = name + ".css";
 
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/stylesheets.spec.ts:11]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/templates.spec.ts [hunk 23]
```diff
@@ -3,7 +3,8 @@ import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
 import {TemplateBuilder} from "@umbraco/json-models-builders";
 
 test.describe('Templates', () => {
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/templates.spec.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Tabs/tabs.spec.ts [hunk 24]
```diff
@@ -6,8 +6,9 @@ const tabsDocTypeName = 'Tabs Test Document';
 const tabsDocTypeAlias = AliasHelper.toAlias(tabsDocTypeName);
 
 test.describe('Tabs', () => {
-  
-  test.beforeEach(async ({ umbracoApi, page }) => {
+
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Tabs/tabs.spec.ts:11]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Tour/tours.spec.ts [hunk 25]
```diff
@@ -3,7 +3,8 @@ import {test} from '@umbraco/playwright-testhelpers';
 
 test.describe('Tours', () => {
   const timeout = 60000;
-  test.beforeEach(async ({page, umbracoApi}) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Tour/tours.spec.ts:7]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/translation.spec.ts [hunk 26]
```diff
@@ -2,10 +2,8 @@ import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
 
 test.describe('Translation', () => {
 
-  test.beforeEach(async ({page, umbracoApi}) => {
-    // TODO: REMOVE THIS WHEN SQLITE IS FIXED
-    // Wait so we don't bombard the API
-    await page.waitForTimeout(1000);
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/translation.spec.ts:6]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Users/userGroups.spec.ts [hunk 27]
```diff
@@ -9,7 +9,8 @@ test.describe('User groups', () => {
     await page.locator('[data-element="sub-view-userGroups"]').click();
   }
 
-  test.beforeEach(async ({ umbracoApi, page }) => {
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Users/userGroups.spec.ts:13]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Users/users.spec.ts [hunk 28]
```diff
@@ -22,8 +22,9 @@ test.describe('Users', () => {
       "userGroups": userGroups,
       "message": ""
     };
-  
-  test.beforeEach(async ({ umbracoApi, page }) => {
+
+  test.beforeEach(async ({ page, umbracoApi }, testInfo) => {
+    await umbracoApi.report.report(testInfo)
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Users/users.spec.ts:27]:
```suggestion
    await umbracoApi.report.report(testInfo);
```

**Comment by andr317c** [comment]
After you merged fram v11/dev the test from the blocklisteditor was added! And they also need the Report!

**Comment by andr317c** [comment]
LGTM! 💪

**Pull request merged**